clang/lib/InstallAPI/DylibVerifier.cpp

```diff
@@ -973,5 +978,24 @@ bool DylibVerifier::verifyBinaryAttrs(const ArrayRef<Target> ProvidedTargets,
   return true;
 }
 
+std::unique_ptr<SymbolSet> DylibVerifier::getExports() {
+  for (const auto &[Alias, Base] : Aliases) {
```

Comment:
This loop runs every time someone calls `getExports()`. Could this be cached, so it only happens once?

Reply:
`getExports()` only gets called once for the whole lifetime of InstallAPI as it releases ownership over the exports to forward along to write out the final TBD file.

Reply:
In that case I would update the function name to `takeExports`.